src/tablator/main.cxx

```diff
@@ -52,13 +72,22 @@ int main(int argc, char *argv[]) {
             std::cout << usage(visible_options) << "\n";
             return 1;
         }
-        if (option_variables.count("stream-intermediate"))
-            stream_intermediate = option_variables["stream-intermediate"].as<bool>();
-        if (option_variables.count("input-format"))
-            input_format.init(option_variables["input-format"].as<std::string>());
+        if (option_variables.count("input-format")) input_format.init(input_format_str);
         if (option_variables.count("output-format"))
-            output_format.init(option_variables["output-format"].as<std::string>());
+            output_format.init(output_format_str);
+        if (option_variables.count("start-row")) {
+            if (!option_variables.count("row-count")) {
+                row_count = 1;
+            }
+        }
+        if (option_variables.count("row-list")) {
+            std::stringstream row_stream(row_string);
+            std::vector<size_t> temp_list((std::istream_iterator<size_t>(row_stream)),
+                                          std::istream_iterator<size_t>());
+            row_list.swap(temp_list);
+        }
```

Comment:
Nitpicking, but shouldn't `start-row` and `row-list` be mutually exclusive? And `row-count` shouldn't be allowed without `start-row`, right?